qcodes/utils/path_helpers.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+QCODES_USER_PATH_ENV = "QCODES_USER_PATH"
+
+
+def get_qcodes_path(*subfolder: str) -> str:
+    """
+    Return full file path of the QCoDeS module. Additional arguments will be
+    appended as subfolder.
+
+    """
+    import qcodes
```

Comment:
Replace this with  https://importlib-resources.readthedocs.io/en/latest/ to avoid depending on qcodes here

Reply:
```
str(importlib_resources.files('qcodes').joinpath(*subfolders)
```